blocks/components/editable/index.js

```diff
@@ -57,6 +59,28 @@ export default class Editable extends wp.element.Component {
 		this.props.onChange( value );
 	}
 
+	onKeyDown( event ) {
+		if ( ! this.props.tagName && event.keyCode === 13 ) {
+			// Wait for the event to propagate
+			setTimeout( () => {
+				// Getting the content before and after the cursor
+				this.editor.selection.getStart();
```

Comment:
Also... what is this line achieving if It's not being assigned? Are there side effects to calling `getStart`?